skycultures/russian_siberian/index.json

```diff
@@ -7,21 +7,21 @@
     {
       "id": "CON russian_siberian Elk",
       "lines": [[67301, 65378, 62956, 59774, 58001, 53910, 54061, 59774]],
-      "common_name": {"english": "Elk", "native": "Лось"}
+      "common_name": {"english": "Elk", "native": "Лось", "pronounce": "ɫ̪os̪ʲ"}
     },
     {
       "id": "CON russian_siberian Kic",
       "lines": [[26241, 26268, 26727, 26311, 25930]],
-      "common_name": {"english": "Threshers", "native": "Кичиги"}
+      "common_name": {"english": "Threshers", "native": "Кичиги", "pronounce": "k̟ʲɪˈt̠̪͡ʲɕ̪iɡ̟ʲɪ"}
     },
     {
       "id": "CON russian_siberian DNe",
       "lines": [[17499, 17489, 17531, 17579, 17664, 17851, 17847, 17608, 17499]],
-      "common_name": {"english": "Duck Nest", "native": "Утиное гнездо"}
+      "common_name": {"english": "Duck Nest", "native": "Утиное гнездо", "pronounce": "uˈt̪ʲin̪ə.jə ɡn̪ʲɪˈz̪d̪o"}
     }
   ],
   "common_names": {
     "HIP 11767": [{"english": "Golden stake"}],
-    "NAME Venus": [{"english": "Night Star", "native": "Ночная звёздочка", "translators_comments": "Native name of Venus in Siberian culture"}]
+    "NAME Venus": [{"english": "Night Star", "native": "Ночная звёздочка", "pronounce": "n̪əˈt̠̪͡ʲɕ̪n̪a.jə ˈz̪vʲo.zd̪ə.t̠̪͡ʲɕ̪kə", "translators_comments": "Native name of Venus in Siberian culture"}]
```

Comment:
Obvious fix: Nochnaja zvjezdochka

Reply:
And you get the ugliness of the "deyofication": it's zvyozdachka, if we are going by pronunciation and not (vulgarized) spelling.

Reply:
Nochnaya zvyozdochka